✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✔ initialize X.Xs
│ ✔ resolving module ref X.Xs
│ ✔ installing module X.Xs
│ ✔ analyzing module X.Xs

✔ prepare X.Xs

✔ viztest: Viztest! X.Xs
✘ .failLog: Void X.Xs
! call function "FailLog": process "/runtime" did not complete successfully: exit code: 2
│ ✘ exec /runtime X.Xs
│ ! process "/runtime" did not complete successfully: exit code: 2
│ 
│ ✔ container: Container! X.Xs
│ $ .from(address: "alpine"): Container! X.Xs CACHED
│ ✔ .withEnvVariable(name: "NOW", value: "20XX-XX-XX XX:XX:XX.XXXX +XXXX UTC m=+X.X"): Container! X.Xs
│ ✘ .withExec(args: ["sh", "-c", "echo im doing a lot of work; echo and then failing; exit 1"]): Container! X.Xs
│ ┃ im doing a lot of work
│ ┃ and then failing
│ │ ✘ exec sh -c echo im doing a lot of work; echo and then failing; exit 1 X.Xs
│ │ ! process "sh -c echo im doing a lot of work; echo and then failing; exit 1" did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide: export NOTHANKS=1

Error logs:

✘ .withExec(args: ["sh", "-c", "echo im doing a lot of work; echo and then failing; exit 1"]): Container! X.Xs
im doing a lot of work
and then failing
